src/main.rs

```diff
@@ -111,7 +115,17 @@ fn main() -> Result<()> {
     }
     drop(toml);
 
-    let mut shell_process = process::Command::new(get_shell());
+    let mut shell_process = match config.editor {
+        None => process::Command::new(get_shell()),
+        Some(editor) => {
+            let mut ide_process = process::Command::new(editor);
+            if let Some(args) = config.editor_args {
+                ide_process.args(args);
+            }
```

Comment:
```suggestion
            ide_process.args(config.editor_args.iter().flatten());
```
I think you can do this